ui/k-info-panel/switch_info.kytos

```diff
@@ -20,6 +20,14 @@
       <k-accordion-item title="Flows" v-if="this.flows">
          <k-flow :content="flow" :key="flow.id" v-for="flow in this.flows"></k-flow>
       </k-accordion-item>
+      <k-accordion-item title="Metadata actions">
+         <div class="metadata_container">
+              <k-textarea title="Insert metadata" icon="arrow-right" placeholder='Eg. {"bandwidth": 100, "port_name": "some_name"}' :value.sync="to_add"></k-textarea>
+              <k-button title="Add metadata" :on_click="bt_add_metadata"></k-button>
+              <k-textarea title="Insert metadata" icon="arrow-right" placeholder="Eg. port_name" :value.sync="to_delete"></k-textarea>
```

Comment:
@Alopalao I think @rmotitsuki meant to use `k-textarea` only when there are multiple inputs like for inserting, here when it comes to deletion it's only a single key, could you replace this component with a `k-input`? Also if you could also update the `title` props it says `Insert metadata`, I think this doesn't get displayed though.

Reply:
Changed on commit [948fff438820ac5e2bc198549045e0de39cc3f44](https://github.com/kytos-ng/topology/pull/74/commits/948fff438820ac5e2bc198549045e0de39cc3f44)